early_stopping/strategies/simple_patience.py

```diff
@@ -0,0 +1,19 @@
+from .linear_patience import LinearPatienceStrategy
+
+
+class SimplePatienceStrategy(LinearPatienceStrategy):
+    """
+    Patience Equation: p(x) = p
+    """
+
+    _name = "simple_patience"
+    _short_name = "SP"
+
+    def __init__(self, b: int = 0, **kwargs):
+        super().__init__(b=b, **kwargs)
```

Comment:
Nothing is stopping someone from specifying `a`.

Instead, `assert "a" not in kwargs`